spec/classes/candlepin_spec.rb

```diff
@@ -169,7 +169,7 @@
         it { is_expected.to compile.with_all_deps }
         it do
           is_expected.to contain_file("/etc/tomcat/server.xml").
-            with_content(/sslProtocols="TLSv1.2,TLSv1.3"/).
+            with_content(/sslProtocol="TLSv1.2+TLSv1.3"/).
```

Comment:
```suggestion
            with_content(/sslProtocol="TLSv1.2,TLSv1.3"/).
```

The comma was correct all along because we use the JSSE style configuration.